samples/drivers/rtc/ds1307/prj.conf

```diff
@@ -0,0 +1,12 @@
+# Enable Peripheral
+CONFIG_I2C=y
```

Comment:
CONFIG_I2C=y is automatically selected by the `RTC_DS1307` device driver, so can be removed